src/argilla/client/feedback/dataset/remote/base.py

```diff
@@ -248,6 +198,7 @@ def __init__(
         fields: List["AllowedFieldTypes"],
         questions: List["AllowedQuestionTypes"],
         guidelines: Optional[str] = None,
+        filters: Dict[str, Any] = {},
```

Comment:
Since this `filters` argument will be used only for FilteredFeedbackDatase, can we move from the base and move to the proper class?  something like this:

```python
class BaseClass:
   pass

class AClass(BaseClass):
  def __init__(...):
      self._records = RemoteRecordsClass()

class BClass(BaseClass):
  def __init__(..., filters):
      super().__init__(...)
      self._records = FiltteredRemoteRecords(filters)

```